packages/calcite-components/src/components/card/card.scss

```diff
@@ -3,35 +3,23 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-card-accent-color-selected: Specifies the accent color of the component when `selected`.
  * @prop --calcite-card-background-color: Specifies the background color of the component.
  * @prop --calcite-card-border-color: Specifies the border color of the component.
- * @prop --calcite-card-shadow: Specifies the shadow of the component.
  * @prop --calcite-card-corner-radius: Specifies the corner radius of the component.
- * @prop --calcite-card-accent-color-selected: Specifies the accent color of the component when `selected`.
- * @prop --calcite-card-selection-background-color: Specifies the background color of the component's selection element.
- * @prop --calcite-card-selection-background-color-hover: Specifies the background color of the component's selection element when hovered.
- * @prop --calcite-card-selection-background-color-active: Specifies the background color of the component's selection element when active.
- * @prop --calcite-card-selection-background-color-selected: Specifies the icon color of the component's selection element when `selected`.
- * @prop --calcite-card-selection-icon-color: Specifies the icon color of the component's selection element.
- * @prop --calcite-card-selection-icon-color-hover: Specifies the icon color of the component's selection element when hovered.
- * @prop --calcite-card-selection-icon-color-selected: Specifies the icon color of the component's selection element when `selected`.
+ * @prop --calcite-card-selection-background-color-active: [Deprecated] Specifies the background color of the component's selection element when active.
+ * @prop --calcite-card-selection-background-color-hover: [Deprecated] Specifies the background color of the component's selection element when hovered.
+ * @prop --calcite-card-selection-background-color-selected: [Deprecated] Specifies the icon color of the component's selection element when `selected`.
+ * @prop --calcite-card-selection-background-color: [Deprecated] Specifies the background color of the component's selection element.
```

Comment:
I’m still not sure why we are removing most of these. How would a user change the color of this internally rendered element? Applies to most of these deprecations.

Reply:
It prevents what is a very common use case for tokens in my mind, something like this:
<img width="723" alt="Screenshot 2024-08-26 at 9 09 46 AM" src="https://github.com/user-attachments/assets/d4f71587-b30b-4214-afc6-e21dc91d2231">

Reply:
these are background colors are for for transparent design pattern behind icons that creates a shading effect. Unless there is a specific user request edge case to break this design pattern there is no need to provide these tokens.

Reply:
I think those underlying values should probably use fg-2, fg-3, not transparent bg. I believe that may have been a holdover from when they were positioned over a thumbnail (?). 

As a designer for the above design though, I'd want to be able to control those.

Reply:
I would agree we should use fg-2, fg-3, not -color-transparent & having control over the above seems reasonable.